#! /bin/sh /usr/share/dpatch/dpatch-run
## 17tutorial_typos.dpatch
##
## DP: Fix typos in the nvi tutorial.

@DPATCH@
--- nvi-1.81.6.orig/nvi-1.79/docs/tutorial/vi.beginner
+++ nvi-1.81.6/nvi-1.79/docs/tutorial/vi.beginner
@@ -505,7 +505,7 @@
 j - moves the cursor DOWN one line
 h - moves the cursor one character to the LEFT
 
-Section 22: {i} {a} {I} {A} {o} {O} ^[ (escape key)
+Section 22: {i} {a} {I} {A} {o} {O} {^[} (escape key)
 
 For this and following sections you will need to use the ESCAPE key on your
 terminal.  It is usually marked ESC.  Since the escape key is the same as
@@ -525,9 +525,9 @@
 on the word 'of'.)
 
 Now carefully type the following string and observe the effects:
-  {iexample ^[}  (remember: ^[ is the escape key)}
+  {iexample ^[}  (remember: ^[ is the escape key)
 The {i} begins the insert mode, and 'example ' is inserted into the line: 
-be sure to notice the blank in 'example '.  The ^[ ends insertion mode, 
+be sure to notice the blank in 'example '.  The {^[} ends insertion mode, 
 and the line is updated to include the new string.  Line 1 should look exactly 
 like Line 2.
 
@@ -541,9 +541,9 @@
 of 'line'.  You can do this in several ways, one way is the following:
 First, type {/line /^M}.  This puts us on the word 'line' in Line 4
 (the blank in the search string is important!).  Next, type {e}.  The 'e' puts
-us at the end of the word.  Now, type {as^[  (^[ is the escape character)}.  
+us at the end of the word.  Now, type {as^[} (^[ is the escape character).  
 The 'a' puts us in insert mode, AFTER the current character.  We appended the 
-'s', and the escape ^[ ended the insert mode.
+'s', and the escape '^[' ended the insert mode.
 
 The difference between {i} (insert) and {a} (append) is that {i} begins
 inserting text BEFORE the cursor, and {a} begins inserting AFTER the cursor.
--- nvi-1.81.6.orig/nvi-1.79/docs/tutorial/vi.advanced
+++ nvi-1.81.6/nvi-1.79/docs/tutorial/vi.advanced
@@ -26,8 +26,8 @@
     1         introduction: {^F} {ZZ}
     2         introduction (con't) and positioning: {^F} {^B}
     3         introduction (con't) and positioning: {^F} {^B}
-    4         positioning: {^F} {^B} ^M (return key)
-    5         quitting: {:q!} ^M key
+    4         positioning: {^F} {^B} {^M} (return key)
+    5         quitting: {:q!} {^M} (return key)
     6         marking, cursor and screen positioning: {m} {G} {'} {z}
     7         marking, cursor and screen positioning: {m} {G} {'} {z}
     8         marking, cursor and screen positioning: {z} {m} {'}
@@ -44,7 +44,7 @@
    19         screen positioning: {H} {M} {L}
    20         character positioning: {w} {b} {0} {W} {B} {e} {E} {'} {`}
    21         cursor positioning: {l} {k} {j} {h}
-   22         adding text: {i} {a} {I} {A} {o} {O} ^[ (escape key)
+   22         adding text: {i} {a} {I} {A} {o} {O} {^[} (escape key)
    23         character manipulation: {f} {x} {X} {w} {l} {r} {R} {s} {S} {J}
    24         undo: {u} {U}
    25         review
